system_tests/tls_system_test.go

```diff
@@ -0,0 +1,187 @@
+package system_tests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	rabbitmqv1beta1 "github.com/rabbitmq/cluster-operator/api/v1beta1"
+	topology "github.com/rabbitmq/messaging-topology-operator/api/v1alpha2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Management over TLS", func() {
+	var (
+		namespace        = MustHaveEnv("NAMESPACE")
+		ctx              = context.Background()
+		targetCluster    *rabbitmqv1beta1.RabbitmqCluster
+		targetClusterRef topology.RabbitmqClusterReference
+		exchange         topology.Exchange
+		policy           topology.Policy
+		queue            topology.Queue
+		user             topology.User
+		vhost            topology.Vhost
```

Comment:
I would suggest test on only one CRD. I would recommend testing on all CRDs if there were no controller test, but you did add tests for all controllers already. I would be very happy just test on one CRD since the logic it's shared between all.